memestra/talk.rst

```diff
@@ -0,0 +1,155 @@
+:title: Gast, Beniget and Memestra
+:data-transition-duration: 150
+:skip-help: true
+:slide-numbers: true
+:css: font.css
+
+
+Intro
+=====
+
+Python is a dynamic language
+
+*Yet* people wants static checks
+
+Could we develop static tooling for Python?
+
+
+----
+
+Prior Works
+===========
+
+``pyflakes``, ``pylint``, ``mypy`` …
+
+Detect unused imports, variable shadowing, undefined variables, style issues,
+type inference based on type hints…
+
+----
+
+Ar Tri Kompagnun
+================
+
+- ``gast``: abstraction of the ``ast`` package across Python versions
+
+- ``beniget``: define - use chains computation for any Python program
+
+- ``memestra``: deprecate use analysis for Python and Jupyter Notebooks
+
+----
+
+Dependency Graphs
+=================
+
+- ``gast`` is used by ``pythran``, ``tangent``, ``beniget``…
+
+- ``beniget`` is used by ``pythran``, ``memestra``, ``transonic``
+
+----
+
+``memestra``
+============
+
+Original goal: detect use of deprecated definitions flagged with a decorator in Python modules
```

Comment:
I know you “only” talk about static check, but might make sense to mention here how deprecation are “usually” handled in the python world (`DeprecationWarning` and `PendingDeprecationWarning`)